src/node_dotenv.cc

```diff
@@ -108,10 +108,14 @@ bool Dotenv::ParsePath(const std::string_view path) {
 }
 
 void Dotenv::AssignNodeOptionsIfAvailable(std::string* node_options) {
-  auto match = store_.find("NODE_OPTIONS");
+  AssignValueIfAvailable("NODE_OPTIONS", node_options);
```

Comment:
We don't need this function anymore. Ref: `AssignNodeOptionsIfAvailable`

Reply:
got it, fixed